user-operator/src/test/java/io/strimzi/operator/user/operator/KafkaUserQuotasIT.java

```diff
@@ -413,13 +408,10 @@ private boolean isPathExist(String path) {
     }
 
     private String encodeUsername(String username) {
-        try {
-            return URLEncoder.encode(username, "UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            throw new RuntimeException("Failed to encode username", e);
-        }
+        return URLEncoder.encode(username, StandardCharsets.UTF_8);
     }
 
+    @SuppressWarnings("SameParameterValue")
```

Comment:
Why is this needed?

Reply:
To stop IDEA nagging me that a method is always called with the same parameter value. I could've used this for some changes several comments above.